src/triage.python/worker.py

```diff
@@ -0,0 +1,375 @@
+#!/usr/bin/env python
+
+# Licensed to the .NET Foundation under one or more agreements.
+# The .NET Foundation licenses this file to you under the MIT license.
+# See the LICENSE file in the project root for more information.
+
+import os
+import urllib
+import zipfile
+import sys
+import site
+import glob
+import shutil
+import json
+
+from time import sleep
+from datetime import datetime
+from os import path
+
+from azure.servicebus   import ServiceBusService, Message, Queue
+from azure.storage      import CloudStorageAccount
+
+_thisFolder = path.dirname(path.realpath(__file__))
+
+# This is our config file. It houses Azure Secrets and other machine dependent paths.
+# It is expected that the config.json sits next to THIS python file.
+_configPath = path.join(_thisFolder, 'config.json')
+if not path.exists(_configPath):
+    FAILURE('expected a file "%s" to exist.' % _configPath)
+
+# used for print debugging
+CONFIG_VERBOSE_OUTPUT = True
+CONFIG_INFORMATIVE_OUTPUT = True
+
+def VERBOSE(value):
+    if CONFIG_VERBOSE_OUTPUT:
+        print(str(datetime.now()) + ' VERBOSE: ' + value)
+
+def INFORMATIVE(value):
+    if CONFIG_INFORMATIVE_OUTPUT:
+        print(str(datetime.now()) + ' INFORMATIVE: ' + value)
+
+def FAILURE(value, error_code = 1, leave = True):
+    print(str(datetime.now()) + ' FAILURE: ' + value)
+    if leave:
+        exit(error_code)
+    
+####################################
+
+class CONFIG(object):
+    def __init__(self, jsonText):
+        self.__dict__ = json.loads(jsonText)
+        
+with open(str(_configPath), 'r') as configFile:
+    data = configFile.read().replace('\n', '')
+    
+_config = CONFIG(data)    
+
+###########################################
+
+# Verify that the paths we DEPEND on for normal operations are in existence.
+# These files MUST exist at the start of the script, AND MUST be absolute paths.
+_path_check_succeeded = True;
+_dependent_paths = { 
+            'triage_ini':           path.join(_thisFolder, 'triage.ini'),
+            'analysis_path':        path.join(_thisFolder, 'analysis.py'),
+            'sos_plugin_path':      str(_config.CONFIG_SOS_PLUGIN_PATH),
+            'lldb_py_path':         str(_config.CONFIG_LLDB_PYTHON_PATH)
+        }
+
+def PATH(key):
+    if _dependent_paths.has_key(key):
+        return str(_dependent_paths[key])
+    else:
+        FAILURE('the key "%s" does not exist as a dependable path. - exiting.' % key, 2)
+
+for key, value in _dependent_paths.iteritems():
+    status = str(key) + ' (' + str(value) + ') ' + ' EXISTS: ' + str(path.exists(value)) + ' IS_ABSOLUTE: ' + str(path.isabs(value))
+    if path.exists(value) and path.isabs(value):
+        INFORMATIVE(status)
+        continue
+    else:
+        _path_check_succeeded = False
+        FAILURE(status, 4)    
+        
+if not _path_check_succeeded:
+    FAILURE('configured paths need to exist, and must be absolute.', 8)
+
+site.addsitedir(PATH('lldb_py_path'))
```

Comment:
Is this code at the root level or is the indentation messed up somehow?  If it's intentional it's a bit confusing